.github/advanced-issue-labeler.yml

```diff
@@ -0,0 +1,76 @@
+# NOTE: This is a common file that is overwritten by realm/ci-actions sync service
+# and should only be modified in that repository.
+
+# syntax - https://github.com/redhat-plumbers-in-action/advanced-issue-labeler#policy
+# Below keys map from the option used in issue form dropdowns to issue labels
+# Limitation:
+# Currently it's not possible to use strings containing ,␣ in the dropdown menus in the issue forms. 
+
+---
+
+policy:
+  - template: [bug.yml, feature.yml]
+    section:
+      - id: [frequency]
+        label:
+          - name: 'Frequency:Once'
+            keys: ['Once']
+          - name: 'Frequency:Sometimes'
+            keys: ['Sometimes']
+          - name: 'Frequency:Always'
+            keys: ['Always']
+
+      - id: [repro]
+        label:
+          - name: 'Repro:Always'
+            keys: ['Always']
+          - name: 'Repro:Sometimes'
+            keys: ['Sometimes']
+          - name: 'Repro:No'
+            keys: ['No']
+
+      - id: [sync, flavour, services]
+        block-list: []
+        label:
+          - name: 'SDK-Use:Local'
+            keys: ['Local Database only']
+          - name: 'SDK-Use:Sync' 
+            keys: ['Atlas Device Sync']
+          - name: 'SDK-Use:Services'
+            keys: ['Atlas App Services: Function or GraphQL or DataAPI etc']
+          - name: ['SDK-Use:All']
+            keys: ['Both Atlas Device Sync and Atlas App Services']
+
+      - id: [encryption]
+        block-list: []
+        label:
+          - name: 'Encryption:On'
+            keys: ['Yes']
+          - name: 'Encryption:Off'
+            keys: ['No']
+            
+      - id: [app-type]
+        block-list: []
+        label:
```

Comment:
in a RealmSwift context, most of the app don't enter in any of this types. Is this relevant to other SDKs and that's why it is here?

Reply:
yes, this is only relevant for .NET. But we use the same common config file. It will have no effect for Swift.